CHANGELOG.md

```diff
@@ -115,20 +118,26 @@ Main (unreleased)
   added extra an extra `|` character when displaying the source file on the
   starting line. (@rfratto)
 
-- Flow: fix issues in `river fmt` where adding an inline comment on the same
+- Flow: fix issues in `agent fmt` where adding an inline comment on the same
   line as a `[` or `{` would cause indentation issues on subsequent lines.
   (@rfratto)
 
-- Flow: fix issues in `river fmt` where line comments in arrays would be given
+- Flow: fix issues in `agent fmt` where line comments in arrays would be given
   the wrong identation level. (@rfratto)
 
 - Flow: fix issues with `loki.file` and `loki.process` where deadlock contention or
   logs fail to process. (@mattdurham)
 
+- Flow: `oath2 > tls_config` was documented as a block but coded incorrectly as
```

Comment:
```suggestion
- Flow: `oauth2 > tls_config` was documented as a block but coded incorrectly as
```

Reply:
Crap, just a second too late. @erikbaranowski can you fix this in the PR to cut the version number?

Reply:
Just saw the thing go purple. 😬 No biggie though